### PR TITLE
new commit with auth controller and ceremony notes from last meeting

### PR DESCRIPTION
**Pull request by jhlbolsonaro**
Two changes:
1. pushing the ceremony notes from our scrum folder for the last meeting we had
2. finished adding the authorization controller, but the register function is still not complete since it needs the SaveUser function that I believe aaron is working on. It has a placeholder for now, but will be complete asap.

### CODE REVIEW

File: tests/auth_controller_test.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import pytest
```
Comment (treyhuculak) [inline tests/auth_controller_test.py]: Tests cover the test cases you need both negative and positive. Well done

**Review by treyhuculak — APPROVED** [review]
Approved to merge
